BUILDING.txt

```diff
@@ -21,6 +23,7 @@ Build Requirements (All Systems)
 
 -- If building RSA-AES support:
    * Nettle 3.0 or later
+   * gmp
```

Comment:
Isn't this just a dependency of nettle? I'm not too fond of duplicating the dependency list recursively for everything we need, as that will not be maintained properly. :/

Reply:
Seems like you are correct, will remove.